File: easy/argentina/argentina.py
def buy(pesos,dollars):

	pesos_dollar = pesos/50

	if pesos_dollar > dollars: 
		print("Dollars")	
	else:
		print("Pesos")


price_dollars = int(input('Price in dollars'))
price_pesos = int(input('Price in pesos'))

buy(price_pesos,price_dollars)

File: easy/argentina/argentina.kt
fun buy(pesos:Int,dollars:Int):String{
    
	if(pesos/50 > dollars){
        return("Dollars")
    }

    return("Pesos")
    
}	


fun main(args: Array<String>){
	
    println("Pesos : ")
    //var pesos = readLine()!!.toInt()

    println("Dollars : ")
    //var dollars = readLine()!!.toInt()
    
    println(buy(pesos,dollars))     
   
  
}
